
---
bool answer_received
int16 level
